readme.md

```diff
@@ -23,6 +23,10 @@ Requires macOS 10.13 or later.
 [![](https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/macappstore-lrg.svg)](https://itunes.apple.com/no/app/gifski/id1351639930?mt=12)
 
 
+## Resizing GIF using keyboard
+
+Press the arrow up/down keys to change the value by 1. Hold the Option key meanwhile to change it by 10.
```

Comment:
This might use a copy change + maybe a screenshot? Or a GIF?

Reply:
A GIF would show it off the best, but I don't know any good ways to make a high-quality GIF...

Reply:
Would you be up for making a GIF? For some reason quick time capture doesn't work for me on macOS Catalina 😢

Reply:
Sure, but let's wait until you've finished the validation logic, so I can show off the bouncing too when trying to increase to a higher number than allowed.